plugins/view-secret.yaml

```diff
@@ -3,19 +3,43 @@ kind: Plugin
 metadata:
   name: view-secret
 spec:
+  version: "v0.1.0"
+  homepage: https://github.com/elsesiy/kubectl-view-secret
+  shortDescription: |
+    Kubernetes CLI plugin to decode Kubernetes secrets
```

Comment:
Everything is a plugin in this repo & on krew search output. :)

Reply:
Done!